src/data/nat/choose/central.lean

```diff
@@ -86,7 +86,7 @@ end
 
 /--
 An exponential lower bound on the central binomial coefficient.
-This bound is weaker than `four_pow_n_lt_n_mul_central_binom`, but it is of historical interest
+This bound is weaker than `four_pow_lt_mul_central_binom`, but it is of historical interest
```

Comment:
Is it in the `nat` namespace? Probably, `doc-gen` won't generate a link without a namespace.